### ISSUE TITLE
Trained weights for ensemble method are missing

### ISSUE THREAD

**Issue by igorRadic** [phamquiluan/ResidualMaskingNetwork#46]
Can you add  "resnet50_pretrained_vgg_rot30_2019Nov13_08.20" and "resnet18_rot30_2019Nov05_17.44" trained weights on your shared google drive folder for ensemble method, they are missing and I can't get 76.82% accuracy without them. Thanks!

**Comment by phamquiluan** [comment]
Thanks, @igorRadic or your interest! My ex-university disable my google drive, and my ex-machine just broke last month. So I technically lost the weights 😞 

I will try to find it in another old machine of mine. Please wait.

**Comment by igorRadic** [comment]
Ok, thank you for your effort! 🙂

**Comment by igorRadic** [comment]
@phamquiluan have you perhaps found trained weights?

**Comment by phamquiluan** [comment]
@igorRadic I can't so far. The weights are not in the old machine. Hmm, I will keep an eye on this matter and return if I find it.

**Comment by igorRadic** [comment]
@phamquiluan  Maybe I can recreate those two models whose weights are missing, but for recreating  "resnet50_pretrained_vgg_rot30_2019Nov13_08.20" I need "resnet50_scratch_dims_2048.pth" weights or not? That's what I assumed because in  "resnet50_pretrained_vgg_rot30_2019Nov13_08.20" there is "pretrained" label and in `models/resnet50_scratch_dims_2048.py` there is commented code for loading weights:
```
569      class Resnet50_pretrained_vgg(Resnet50_scratch):
570          def __init__(self):
571              super(Resnet50_pretrained_vgg, self).__init__()
572              # state_dict = torch.load('./saved/pretrained/resnet50_scratch_dims_2048.pth')
573              # self.load_state_dict(state_dict)
```
also i don't understand why is in this model name 'vgg', can you please explain how to recreate "resnet50_pretrained_vgg_rot30_2019Nov13_08.20" model? Thanks!

**Comment by phamquiluan** [comment]
Wow, you truly impressed me with your determination, @igorRadic. I conducted this research for a long time and forgot almost everything detail about the naming and conventions of using which model and not.

I assume that you are a researcher. Please read this email, the screenshot I attached. I hope this help!

<img width=400 src="https://user-images.githubusercontent.com/24642166/227734704-354181ff-28c7-4dc1-adec-0470e8a1cab6.png" />

p/s: I tried so much to keep the reproducibility when I started this research years ago... I'm mad at myself so much when this training code can't be reproduced anymore and the school delete my drive :(

**Comment by igorRadic** [comment]
@phamquiluan thank you for your quick response and honesty, I will definitely try to reproduce your results and If I make it I will let you know.

**Comment by chn-lee-yumi** [comment]
Hi @igorRadic , have you reproduced the result successfully?

**Comment by igorRadic** [comment]
Hi @chn-lee-yumi , I got 76.34 % test accuracy without these two weights.

**Comment by chn-lee-yumi** [comment]
> I got 76.34 % test accuracy without these two weights.

@igorRadic That's weird. Theoretically, we should get the same result. Did you use `gen_results.py` and `gen_ensemble.py` to get the result?

**Comment by igorRadic** [comment]
@chn-lee-yumi Maybe that is because I made a few code fixes, try to reproduce 76.34 % test accuracy with my fork of this repository. And yes, I used `gen_results.py` and `gen_ensemble.py` scripts for reproducing this result.

**Comment by chn-lee-yumi** [comment]
@igorRadic Thanks, I will try your fork. By the way, how did you deal with the missing `test_targets.npy`? I wrote a simple script read from the json file and converted it.

**Comment by igorRadic** [comment]
@chn-lee-yumi Problem with `test_targets.npy` is also fixed in my fork, you can simply download the wights and run `gen_results.py` and `gen_ensemble.py` scripts.

**Comment by chn-lee-yumi** [comment]
@igorRadic Hi, did you archive 74.14% using `fer2013_ResMaskingNet_74.14_test_acc_config.json`? I only got 68.849%.
